### ISSUE TITLE
Bug - Tooltip - contents for ref variant not announced by assistive tech

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#7331]
**Describe the problem**
Currently when using a React or selector ref for the Tooltip component, the tooltip contents does not get announced by assistive technologies. Only using the Tooltip component as a wrapper has the contents announced.

**How do you reproduce the problem?**

1. Go to the [Tooltip component](https://www.patternfly.org/v4/components/tooltip/)
2. Start Voice Over or another screen reader
3. Navigate to the examples that use a React or selector Ref
4. Notice that the tooltip contents may not get announced (tested using VO in Chrome)

**Expected behavior**
The tooltip contents gets announce when using the React or selector ref variants

**Is this issue blocking you?**
An alternative is using the Tooltip component as laid out in the "basic" example in the above link (using the Tooltip component as a wrapper).

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**

**What is your product and what release date are you targeting?**

**Any other information?**
